javascript/schema/src/datatable/ParserOptions.ts

```diff
@@ -56,5 +56,5 @@ export interface ParserOptions {
 	 * Number of rows to read from the data, starting at skip_rows (honors skipBlankLines).
 	 * Default: Infinity
 	 */
-	readRows?: number
+	nRows?: number
```

Comment:
I'm not really a big fan of `nRows`. I realize it is the arg name in pandas, but it feels to me like a formal JSONSchema should be a bit more verbose for clarity.